src/extensions/component-enumerators/examples/contoso-component-enumerator/demo/sample-updates/CreateSampleMSOEUpdate-8.x.ps1

```diff
@@ -244,7 +234,7 @@ Write-Host "Preparing update $parentUpdateIdStr ..."
     # ADD STEP(S)
     #
 
-    # This update contains 3 steps.
+    # This update contains 2 steps.
```

Comment:
maybe remove comment so don't need to keep updating?

Reply:
Removed